mitodl/mitxonline
# add a tracking table for course run purchases

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/mitxonline/issues/738)

#### What's this PR do?
 Add a new table to track when order is fulfilled or in review state. see https://github.com/mitodl/mitxonline/issues/738 for detail
 The insert logic is implemented in the following places
- When order is transition from pending or review to fulfilled
- When order is transition from pending to review

Note that 
- If order is manually updated to fulfilled or review from admin directly, It would not work.
- No delete is implemented

#### How should this be manually tested?
enroll a course from frontend, it should display this if order is paid successful
<img width="1146" alt="image" src="https://user-images.githubusercontent.com/3138890/181248408-d2bd8153-8e65-4f69-8f82-ed859d582c77.png">
Go to admin/courses/paidcourserun/, you should see the fulfilled order shows up there
![image](https://user-images.githubusercontent.com/3138890/181248754-0f700698-85a6-4911-b722-8865f661da07.png)



#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)
